roles/neutron-data-network/tasks/monitoring.yml

```diff
@@ -31,6 +31,9 @@
   sensu_check: name=check-l3-router plugin=check-l3-router.sh 
                args="-e {{ ansible_default_ipv4.interface }}"
                use_sudo=true
+  with_items:
+   - "{{ neutron.networks }}"
```

Comment:
You shouldn't have to put this in brackets, and thus not in quotes. 

`with_items: neutron.networks`